### ISSUE TITLE
Placeholder text color contrast on macOS safari

### ISSUE THREAD

**Issue by CharlesKuncheria** [microsoft/fluentui#19282]
### Environment Information

- **Package version(s)**: (fluent ui react v8 or v7)
- **Browser and OS versions**: (macOS bigsur safari & on safari iOS)

### Describe the issue:

On MacOS safari (or in iOS safari), the placeholder text color doesn't meet a11y requirement of 4:5 : 1
Note: Color contrast is respected on Chrome browser.

### Please provide a reproduction of the issue in a codepen:

Steps:
1. On Mac OS safari go to codepen link https://codepen.io/brendanmckeon/pen/NWjLzBW
2. Notice the placeholder text for the people picker input field.
3. Color is #a9a9a9 on a white background has a color contract ratio of 2:35 : 1

#### Actual behavior:

Current color contrast is 2:35 : 1

#### Expected behavior:

Expected color contract > 4.5 : 1

Similar issue, but nor the same: https://github.com/microsoft/fluentui/issues/17661

**Comment by smhigley** [comment]
It looks to me like our pickers aren't using the same placeholder styles as Input. We should update the styles to be consistent with the Input styles.

The linked issue looks like a red herring; that one is referencing an old Theme Designer link and doesn't repro on current code; picker placeholder color is a separate issue.
